python/adaptivity.py

```diff
@@ -596,7 +596,7 @@ def metric_pnorm(f, eta, max_edge_length=None, min_edge_length=None, max_edge_ra
   # EXTRACT HESSIAN
   [HH,cell2dof] = get_dofs(H)
   # add DOLFIN_EPS on the diagonal to avoid zero eigenvalues
-  HH[0,:] += DOLFIN_EPS
+  HH[0,:] += 1e3*DOLFIN_EPS
```

Comment:
I made a new pull request without this change.

On 01/06/15 11:10, Gerard Gorman wrote:
>
> In python/adaptivity.py 
> <https://github.com/meshadaptation/pragmatic/pull/28#discussion_r31413475>:
>
> > @@ -596,7 +596,7 @@ def metric_pnorm(f, eta, max_edge_length=None, min_edge_length=None, max_edge_ra
> >    # EXTRACT HESSIAN
> >    [HH,cell2dof] = get_dofs(H)
> >    # add DOLFIN_EPS on the diagonal to avoid zero eigenvalues
> > -  HH[0,:] += DOLFIN_EPS
> > +  HH[0,:] += 1e3*DOLFIN_EPS
>
> Why this change to HH?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/meshadaptation/pragmatic/pull/28/files#r31413475>.
>